app/src/main/res/menu/main_menu.xml

```diff
@@ -36,7 +31,7 @@
             <group android:checkableBehavior="single">
                 <item
                     android:id="@+id/nav_edit_profile"
-                    android:icon="@drawable/settings"
+                    android:icon="@drawable/ic_person_black_24dp"
```

Comment:
Is this going to change the icon to access the user profile in the side panel? If yes, then nice, this `ic_person_black_24dp` is more appropriate for accessing the user account!

Reply:
Yes, I also change the name of this panel to Profile, so this new image is more appropriate for this new name